src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -409,7 +409,8 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
     }
 
     this.business = {
-      bcRegistrationDate: this.dateToYyyyMmDd(this.apiToDate(businessInfo.foundingDate)),
+      bcRegistrationDate: businessInfo.foundingDate,
+      bcRegistrationDateFormatted: this.dateToYyyyMmDd(this.apiToDate(businessInfo.foundingDate)),
```

Comment:
I wouldn't add this to the business object.

How about creating a getter (computed) to return this, that you can use in the template and rules above?

Reply:
Updated this, thanks.